web/Services/AdminService.cs

```diff
@@ -136,42 +139,63 @@ public async Task<List<LandlordDbModel>> GetLandlordList(LandlordListModel landl
         return await landlordQuery.ToListAsync();
     }
 
-    public async Task<List<TenantDbModel>> GetTenantList(HousingRequirementModel filters)
+    private IQueryable<TenantDbModel> GetFilteredTenantQuery(HousingRequirementModel filters, bool filterOrMatching)

```

Comment:
So we have all these values as bools and we are then converting them into a list. I am not exactly sure why this is happening here, instead of just using the actual boolean properties on the model with defaults as false, below is an example of how this would look.

Also the variable name `filterOrMatching` doesn't actually tell me whether it is a filter or matching, could you please rename :)

Reply:
Oh and can we please put the private methods below the public methods